stage1/app-rm/app-rm.go

```diff
@@ -86,24 +113,54 @@ func main() {
 
 	for _, p := range appServicePaths {
 		if err := os.Remove(p); err != nil && !os.IsNotExist(err) {
-			log.FatalE("error removing app service file", err)
+			return fmt.Errorf("error removing app service file: %s", err)
 		}
 	}
 
-	args = enterCmd
-	args = append(args, "/usr/bin/systemctl")
-	args = append(args, "daemon-reload")
+	// TODO(sur): find all RW cgroups exposed for this app and clean them up
```

Comment:
@s-urbaniak: this TODO is in place here as a reminder to re-visit the rm steps once https://github.com/coreos/rkt/pull/3389 is merged.